erpnext/shopping_cart/cart.py

```diff
@@ -112,9 +112,11 @@ def update_cart(item_code, qty, with_items=False):
 	quotation.payment_schedule = []
 	if not empty_card:
 		quotation.save()
+		cart_count = cstr(len(quotation.get("items")))
```

Comment:
Is this supposed to be a string? Or did you mean to use `cint`?

Reply:
Good catch, I think cint is better since I'm casting to string in the callback. I just made that change.

Reply:
Ah, no problem. Glad to help.

I had another note: although your way is more explicit, Javascript is a weakly typed language, so you wouldn't need to do the string conversion in order to do the concatenation.

Reply:
True, good point. I'll make that change as well